### PR TITLE
Update ALFRESCO ingests for current Zeus coverages and add style hooks to ingests.

### PR DESCRIPTION
**Pull request by charparr**
This PR updates the ALFRESCO ingests such that they are able to recreate the ALFRESCO coverages that are currently on Zeus and used by the Climate Impact Reports Tool. Some metadata and encoding issues were fixed, more descriptive coverage names were chosen, past / future coverages were combined, and deprecated coverages were archived. Finally, post-import "hooks" to add styles via the `rasdaman/admin/layer/style/add`endpoint were included where appropriate to implement version control for styles and to speed-up the coverage re-ingest process.

There is also an addition of a text file that just lists the files accessed by the ingest - I don't know that this needs to be a permanent addition, but I think it is useful in the interim as a reference until we set up some type of syncing between the ingest data itself and a "do-all-the-ingests.sh" program.

There is also an update to the documentation to reflect what we've learned about pushing styles up to Rasdaman.

### CODE REVIEW

File: iem/alfresco/relative_flammability/README.md [hunk 1]
```diff
@@ -3,23 +3,20 @@
 This subfolder contains the processing notebook and ingest recipes for a relative flammability dataset. These data are created from outputs of the ALFRESCO runs done with CMIP5 outputs for the SERDP Fish and Fire project, and can be found at `/workspace/Shared/Tech_Projects/SERDP_Fish_Fire/project_files/Calibration/HighCalib/FMO_Calibrated`.
```
Comment (kyleredilla) [inline iem/alfresco/relative_flammability/README.md:3]:
```suggestion
This subfolder contains the processing notebook and ingest recipes for a relative flammability dataset. These data are created from outputs of the ALFRESCO runs done with CMIP5 outputs for the IEM project, and can be found at `/workspace/Shared/Tech_Projects/Alaska_IEM/project_data/Final_runs/IEM_AR5`.
```

I forgot to update these a while back when I found that we had these "IEM" ALFRESCO runs! This is what's used in the code.

File: iem/alfresco/relative_flammability/ingest.json [hunk 2]
```diff
@@ -2,14 +2,23 @@
...
+    {
+      "description": "Create Style for WMS Layer",
+      "when": "after_import",
+      "cmd": "curl \"http://localhost:8080/rasdaman/admin/layer/style/add?COVERAGEID=alfresco_relative_flammability_30yr&STYLEID=climate_impact_reports_alfresco_relative_flammability_30yr&COLORTABLETYPE=ColorMap&\" --data-urlencode \"COLORTABLEDEFINITION={\\\"type\\\": \\\"ramp\\\", \\\"colorTable\\\": {  \\\"-9999\\\": [0, 0, 0, 0], \\\"0\\\": [255, 255, 255, 255], \\\"0.03\\\": [255, 0, 0, 255] } }\"",
+      "abort_on_error": true
+    }
+  ],
   "input": {
-    "coverage_id": "relative_vegetation_change_future",
+    "coverage_id": "foo",
```
Comment (kyleredilla) [inline iem/alfresco/relative_flammability/ingest.json:19]: Verify `"coverage_id":` string.

**Review by kyleredilla — CHANGES_REQUESTED** [review]
Changes look great! Just a couple things to check. Appreciate the inclusion of filenames that are used in the coverages in text files, that is a good idea. I think if we ever make a place on Poseidon to backup all tarballs or something of all ingested data, we can include that dir in those text files as well, or collapse it down to a single file with coverage ID -- tarball mappings.

**Comment by charparr** [comment]
I thought I had squashed this locally and pushed it before making the PR using `git reset soft --head` but it doesn't seem like I was very effective. Definitely squash this one I think.

**Review by kyleredilla — APPROVED** [review]

**Pull request merged**